src/sum.js

```diff
@@ -6,13 +6,25 @@
  * @returns {number} The sum
  */
 function sum(array) {
-  // This is your job. :)
+  let sum =0;
```

Comment:
Careful, you've overwritten the `sum` function in the scope here. Use a variable name like `result` or `runningTotal` or `sumSoFar` etc.